apps/vtex-my-subscriptions-3/messages/en.json

```diff
@@ -25,7 +25,7 @@
   "subscription.data.orderAgain": "Reorder:",
   "subscription.data.chargeEvery": "Repeat:",
   "subscription.select": "Select",
-  "subscription.select.day": "Day {day}",
+  "subscription.select.day": "On the {day, selectordinal, one {#st} two {#nd} few {#rd} other {#th}} of the month",
```

Comment:
Would this be the case to the localization team review all other languages?

Reply:
Sure, I'll submit a request to the localization team